enterprise/cli/server_dbcrypt_test.go

```diff
@@ -41,15 +44,38 @@ func TestServerDBCrypt(t *testing.T) {
 	})
 	db := database.New(sqlDB)
 
+	t.Cleanup(func() {
+		if t.Failed() {
+			t.Logf("Dumping data due to failed test. I hope you find what you're looking for!")
```

Comment:
oh, this should be a general pattern in coder/coder. Great idea!

Reply:
Indeed. Actually I'm wondering if it would make sense to have/write a full `psql` dump as well (or instead). It'd be great if the test author didn't need to consider all failure scenarios and the information would always be there in full (and repeatable by importing dump).

Reply:
I can add that as a follow-up PR! Maybe something like `dbtestutil.WithDumpOnFailure(t)`?